lasagne/layers/cuda_convnet.py

```diff
@@ -99,7 +99,9 @@ class Conv2DCCLayer(CCLayer):
         An initializer for the weights of the layer. This should initialize the
         layer weights to a 4D array with shape
         ``(num_filters, num_input_channels, filter_rows, filter_columns)``.
+
          If automatic dimshuffling is disabled (see notes), the shape should be
+
         ``(num_input_channels, input_rows, input_columns, num_filters)``
```

Comment:
> Nice. How did you spot that?

Well, the error message clearly says "Unexpected indentation."

> Does the line number refer to a specific file or?

It refers to the line of the docstring where the error occurs. You need to add the line number of the first docstring line to convert it to file line numbers. It's a bit silly, the error message would be a lot more useful if it just printed the line of the error, or gave the file line number, instead of printing a list of all docstring lines.